apps/condo/domains/ticket/schema/CreateResidentTicketService.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Generated by `createservice ticket.CreateResidentTicketService --type mutations`
+ */
+const { Contact } = require('@condo/domains/contact/utils/serverSchema')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+const { Ticket } = require('../utils/serverSchema')
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/ticket/access/CreateResidentTicketService')
+const { NOT_FOUND_ERROR } = require('@condo/domains/common/constants/errors')
+const { getSectionAndFloorByUnitName } = require('@condo/domains/ticket/utils/unit')
+
+const TICKET_MOBILE_SOURCE_ID = '3068d49a-a45c-4c3a-a02d-ea1a53e1febb'
+
+const CreateResidentTicketService = new GQLCustomSchema('CreateResidentTicketService', {
+    types: [
+        {
+            access: true,
+            type: 'input CreateResidentTicketInput { dv: Int!, sender: JSON!, details: String!, propertyId: String!, unitName: String }',
+        },
+        {
+            access: true,
+            type: 'type ResidentTicketOutput { organization: Organization!, property: Property!, unitName: String,' +
+                'sectionName: String, floorName: String, number: Int!, client: User!, clientName: String,' +
+                'clientEmail: String, clientPhone: String, contact: Contact, operator: User, assignee: User, executor: User,' +
+                'details: String!, related: Ticket, isEmergency: Boolean,' +
+                'isPaid: Boolean, source: TicketSource!, id: String!, createdBy: User!, createdAt: String!,' +
+                'updatedAt: String, updatedBy: User, watchers: [User], classifier: TicketClassifier, meta: JSON, sourceMeta: JSON,' +
```

Comment:
no need to access watchers, no need to access to meta, sourceMeta, updatedBy, createdBy, operator, assignee, executor, contact.